src/index.tsx

```diff
@@ -82,15 +84,13 @@ function Root() {
                     <TextResourcesProvider>
                       <KeepAliveProvider>
                         <WindowTitleProvider>
-                          <DevTools>
-                            <App />
-                            <ToastContainer
-                              position='top-center'
-                              theme='colored'
-                              transition={Slide}
-                              draggable={false}
-                            />
-                          </DevTools>
+                          <App />
```

Comment:
Are we temporarily removing `DevTools`?

Reply:
Nope, it has been moved to `<PresentationComponent />`. That way it is always rendered _inside_ all the providers. It was rendered here as well, possibly because of a borked merge conflict resolution, so it was rendered twice.